app/ports/views.py

```diff
@@ -403,6 +405,8 @@ def maintainer_detail_github(request, github_handle):
 
 def maintainer_detail_email(request, name, domain):
     maintainers = Maintainer.objects.filter(name=name, domain=domain)
+    if not maintainers.count() > 0:
```

Comment:
I think we can write it in a more straightforward way which to me is more readable: `if maintainers.count() == 0`. What do you think?

Reply:
Seems fine to me.